src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,47 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, onTabSelected, selectedTabId }) => {
+  const checkedSelectedIdTable =
+    tabs.find(e => e.id === selectedTabId)?.id || tabs[0].id;
+
+  return (
+    <>
+      <div data-cy="TabsComponent">
+        <div className="tabs is-boxed">
+          <ul>
+            {tabs.map(tab => {
+              return (
+                <>
+                  <li
+                    key={tab.id}
+                    className={cn({
+                      'is-active': checkedSelectedIdTable === tab.id,
+                    })}
+                    data-cy="Tab"
+                  >
+                    <a
+                      href={`#${tab.id}`}
+                      data-cy="TabLink"
+                      onClick={() => {
+                        if (tab.id !== checkedSelectedIdTable) {
+                          onTabSelected(tab);
+                        }
+                      }}
+                    >
+                      {tab.title}
+                    </a>
+                  </li>
```

Comment:
This TabLink code here should be another component for a better reusability, don't you think?